doc/source/tutorial/computeclass.rst

```diff
@@ -0,0 +1,112 @@
+.. _computeclass:
+
+===============
+Compute Classes
+===============
+
+Calculations in **freud** are built around the concept of ``Compute classes``, Python objects that encode a given method and expose it through a ``compute`` method.
+In general, these methods operate on a particular triclinic box, which can be provided as any object that can be interpreted as a **freud** box (as demonstrated in the examples above).
+We can look at the :class:`freud.order.HexOrderParameter` class as an example:
+
+.. code-block:: python
+
+    import freud
+    positions = ...  # Read positions from trajectory file.
+    op = freud.order.HexOrderParameter(k=6)
+    op.compute(
+        ({'Lx': 5, 'Ly': 5, 'dimensions': 2}, positions),
+        dict(r_max=3)
+    )
+
+    # Plot the value of the order parameter.
+    from matplotlib import pyplot as plt
+    plt.hist(np.absolute(op.order))
+
+Here, we are calculating the hexatic order parameter, then using Matplotlib to plot 
+The :class:`freud.order.HexOrderParameter` class constructor accepts a single argument :code:`k`, which represents the periodicity of the calculation.
+If you're unfamiliar with this order parameter, the most important piece of information here is that many compute methods in **freud** require parameters that are provided when the ``Compute class`` is constructed.
+
+To calculate the order parameter we call :meth:`compute <freud.order.HexOrderParameter.compute>`, which takes two arguments, a :class:`tuple` `(box, points)` and a :class:`dict`.
+We first focus on the first argument.
+The ``box`` is any object that can be coerced into a :class:`freud.box.Box` as described in the previous section; in this case, we use a dictionary to specify a square (2-dimensional) box.
+The ``points`` must be anything that can be coerced into a 2-dimensional NumPy array of shape :code:`(N, 3)`
+In general, the points may be provided as anything that can be interpreted as an :math:`N\times 3` list of positions; for more details on valid inputs here, see :func:`numpy.asarray`.
+Note that because the hexatic order parmaeter is designed for two-dimensional systems, the points must be provided of the form `[x, y, 0]` (i.e. the z-component must be 0).
```

Comment:
```suggestion
Note that because the hexatic order parameter is designed for two-dimensional systems, the points must be provided of the form `[x, y, 0]` (i.e. the z-component must be 0).
```